index.html

```diff
@@ -56,6 +56,127 @@ <h1>
         </li>
     </ul>
 </div>
+<div class="readmore-button">
+    <a>
+        <img src="./images/icons/readmore-icon.svg" alt="read more ">
+    </a>
+</div>
+</section>
+<section id="works" >
+    <div class="recent-works">
+        <h2>My Recent </br> Works</h2>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+</section>
```

Comment:
- [ ] Great job on your works section! To align with the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and improve the layout, could you kindly consider using the `grid` display for the parent element of the works section? This will ensure that the section's items are properly arranged and easier to view. 🤓

  ![image](https://user-images.githubusercontent.com/107952724/235261474-b6215fa5-e758-4403-bceb-e6f0abeed515.png)